pdns/dnsdist-cache.hh

```diff
@@ -144,5 +143,5 @@ private:
   bool d_deferrableInsertLock;
   bool d_parseECS;
   bool d_keepStaleData{false};
-  bool d_cookieHashing{false};
+  std::unordered_set<uint16_t> d_optionsToSkip{EDNSOptionCode::COOKIE};
```

Comment:
Mostly my OCD acting up but I'd prefer moving this right after the `d_shards` one :) We don't really care about the size of that struct here but it might also prevent the need for padding.